keep/api/models/db/migrations/versions/2025-03-07-17-49_aaec81b991bd.py

```diff
@@ -0,0 +1,316 @@
+"""Crecreate topology tables with tenant_id in all keys and transfer data
+
+Revision ID: aaec81b991bd
+Revises: 0b80bda47ee2
+Create Date: 2025-03-07 17:49:13.393091
+
+"""
+
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+from sqlalchemy.orm import Session
+
+# revision identifiers, used by Alembic.
+revision = "aaec81b991bd"
+down_revision = "0b80bda47ee2"
+branch_labels = None
+depends_on = None
+
+def transfer_data():
+    session = Session(bind=op.get_bind())
+    dialect = session.bind.dialect.name
+
+    uuid_generation_func = "replace(uuid(),'-','')"
+    if dialect == "sqlite":
+        uuid_generation_func = """
+        lower(
+            hex(randomblob(4)) || '-' || hex(randomblob(2)) || '-' || '4' ||
+            substr(hex( randomblob(2)), 2) || '-' ||
+            substr('AB89', 1 + (abs(random()) % 4) , 1)  ||
+            substr(hex(randomblob(2)), 2) || '-' ||
+            hex(randomblob(6))
+          )
```

Comment:
Oh wow

Reply:
Yeah, but it's the best way I know to generate uuid in sqlite without external code =)